deltachat_derive/src/lib.rs

```diff
@@ -8,36 +8,33 @@ use quote::quote;
 // data.  If this assumption is violated, compiler error will point to
 // generated code, which is not very user-friendly.
 
-#[proc_macro_derive(ToSql)]
-pub fn to_sql_derive(input: TokenStream) -> TokenStream {
+#[proc_macro_derive(Sqlx)]
```

Comment:
IIUC this is only for types that can be changed into i64.  Would it make sense to make the name a bit more specific?  `SqlxInt64`?

Reply:
Maybe